lib/console_web/schema/schema.ex

```diff
@@ -283,6 +303,10 @@ defmodule ConsoleWeb.Schema do
       resolve &Console.Labels.LabelResolver.find/2
     end
 
+    field :all_alerts, list_of(:alert) do
```

Comment:
missing single device schema here?

Reply:
nvm thought you added the show page already

Reply:
not yet hehe